cmp/dataview/DataView.ts

```diff
@@ -22,7 +23,7 @@ import {isFunction, merge} from 'lodash';
 import './DataView.scss';
 import {DataViewModel} from './DataViewModel';
 
-export interface DataViewProps extends HoistProps<DataViewModel>, BoxProps {
+export interface DataViewProps extends HoistProps<DataViewModel>, LayoutProps, TestSupportProps {
```

Comment:
It seems like this does support `BoxProps` still, or am I missing something? `dataView` returns a `grid` which returns an `agGrid` which returns a `frame` which returns a `box`. It seems at first glance like the `BoxProps` are passed down, but I might be overlooking something?